src/components/timer/quiz-timer.js

```diff
@@ -0,0 +1,57 @@
+var timer;
+
+export function createTimer() {
+  const clock = document.createElement('div');
+  clock.setAttribute('id', 'clock');
+
+  const image = document.createElement('img');
+  image.setAttribute('id', 'clk_img');
+  image.src = '/timer.png';
+
+  const minutes = document.createElement('span');
+  minutes.setAttribute('id', 'min');
+  minutes.innerText = '00';
+
+  const colon = document.createElement('span');
+  colon.innerText = ':';
+
+  const seconds = document.createElement('span');
+  seconds.setAttribute('id', 'sec');
+  seconds.innerText = '00';
```

Comment:
To improve readability, replace abbreviations in ids with full names eg. min-> minutes or even timer-minutes